compiler/one-cmds/one-build

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import configparser
+import os
+import subprocess
+import sys
+
+import one_utils as _utils
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to run ONE drivers in customized order')
+
+    _utils._add_default_arg(parser)
+
+    return parser
+
+
+def _parse_arg(parser):
+    args = parser.parse_args()
+    # print version
+    if args.version:
+        _utils._print_version_and_exit(__file__)
+
+    return args
+
+
+def _verify_arg(parser, args):
+    """verify given arguments"""
+    # check if required arguments is given
+    if not _utils._is_valid_attr(args, 'config'):
+        parser.error('-C/--config argument is required')
+
+
+def _get_driver_name(driver_name):
+    return {
+        'one-import-bcq': 'one-import-bcq',
+        'one-import-tf': 'one-import-tf',
+        'one-import-tflite': 'one-import-tflite',
+        'one-optimize': 'one-optimize',
+        'one-pack': 'one-pack',
+        'one-codegen': 'one-codegen'
+    }[driver_name]
+
+
+def _parse_cfg(args):
+    config = configparser.ConfigParser()
+    config.read(getattr(args, 'config'))
+    return config
+
+
+def _is_available_driver(config, driver_name):
+    return config.has_option('driver-to-run', driver_name) and config.getboolean('driver-to-run', driver_name)
+
+
+def _verify_cfg(driver_list, config):
+    if not config.has_section('driver-to-run'):
+        raise AssertionError('driver-to-run section is required in configuraion file')
+
+    import_driver_cnt = 0
+    if _is_available_driver(config, 'one-import-tf'):
+        import_driver_cnt+=1
+    if _is_available_driver(config, 'one-import-tflite'):
+        import_driver_cnt+=1
+    if _is_available_driver(config, 'one-import-bcq'):
+        import_driver_cnt+=1
+    if import_driver_cnt > 1:
+        raise AssertionError('only one import-* driver can be executed')
+
+
+def main():
+    # parse arguments
+    parser = _get_parser()
+    args = _parse_arg(parser)
+
+    # verify arguments
+    _verify_arg(parser, args)
+
+    # parse configuration file
+    config = _parse_cfg(args)
```

Comment:
sequence of `_verify_arg` -> `_parse_cfg`: how does this work if both command line arg and cfg file exist?

Reply:
@seanshpark As we've discussed earlier, arg overrides cfg.

Reply:
Why I asked is funcion call sequence is `_verify_arg` then `_parse_cfg`.
One may think (1) set the `args` with command line (2) override `args` with cfg file.
So, can we swap these?

Reply:
Unfortunately, I can't. Actually, I have to verify `config` - configuration file path - argument  before parsing configuration file. It seems natural...

Reply:
Sorry, not `_verify_arg`...
it is about `_parse_arg` -> `_parse_cfg` sequence.

Reply:
@seanshpark Right. It might seem unnatural as you said. But as with above, conf file path first has to be given before parsing conf file. So I should parse arg first.